timeutil/timeutil.go

```diff
@@ -0,0 +1,40 @@
+package timeutil
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// MinsOfDay returns minutes since start of day for a timestamp in format `15:30`
+// eg. MinsOfDay("02:30") -> 150
+// returns 0 for a timestamp not in format HH:MM
+func MinsOfDay(str string) int {
+	matched, err := regexp.MatchString("^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$", str)
+	if err != nil || !matched {
+		return 0
+	}
+	x := strings.Split(str, ":")
+	hrs, _ := strconv.Atoi(x[0])
+	mins, _ := strconv.Atoi(x[1])
+	return (hrs * 60) + mins
+}
+
+// StartOfDay returns start of the day
+func StartOfDay(t time.Time) time.Time {
+	return time.Date(t.Year(), t.Month(), t.Day(), 0, 0, 0, 0, t.Location())
+}
+
+// Now returns the current time in UTC.
+func Now() time.Time {
```

Comment:
why is `timeutil.Now()` better than `time.Now().UTC()`?

Reply:
All of this copy pasted from rudder-server and these are used by warehouse modules. 

Probably we should rather change in server than move them here for these usecases

Reply:
To me these functions don't seem to be worth having